docs/deployer/container.md

```diff
@@ -70,11 +86,16 @@ In order for the container deployer to interact with the landscaper a contract f
   }
   ```
 
-- The optional *content blob* that can be defined by a definition can be accessed at the directory given by the env var `CONTENT_PATH`.
+- The optional *content blob* that can be defined by a definition can be accessed at the directory given by the env 
```

Comment:
we should add that the content is only given if a `blueprint` in the provider configuration is defined